packages/documentation/src/app/overview/roadmap/page.mdx

```diff
@@ -88,14 +88,6 @@ release. There's a decent chance that many of these will beat a v1 release still
   but also be a forcing function to tidy up the koa template code, and abstract it in
   a similar way to the client templates are.
 
-  ### Complete an interactive playground
```

Comment:
The only change other than dependency / data refresh is removing this completed item from the roadmap.